src/styles/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background: rgb(255, 255, 255);
```

Comment:
Consider using SCSS variables for colors like `rgb(255, 255, 255)` to improve maintainability and consistency across your stylesheets.